src/index.ts

```diff
@@ -10,26 +19,75 @@ export async function sync(name: string, from: Record<string, string>, to: Recor
   // fullMetadata may needed to obtain the repository property in manifest
   srcOptions.fullMetadata = true
   const srcPackument = await getPackument(name, srcOptions)
-  const srcVersions = srcPackument ? Object.keys(srcPackument.versions) : []
-  console.debug('Source versions', srcVersions)
+  const srcVersionsRaw = srcPackument ? Object.keys(srcPackument.versions) : []
+
+  console.debug('Source versions', srcVersionsRaw)
+
+  let srcVersions = srcVersionsRaw;
+
+  if (srcPackument && latestOnly) {
+    srcVersions = [srcPackument['dist-tags'].latest]
+    console.debug('Latest version', srcVersions)
+  } else if (srcVersionsRaw.length && latestMajors) {
+    const removePrereleases = srcVersionsRaw.filter(version => {
+      const semverInstance = new semver.SemVer(version)
+      return semverInstance.prerelease.length === 0;
+    });
+
+    srcVersions = Array.from(semver.rsort(removePrereleases).reduce((acc, version) => {
+      const semverInstance = new semver.SemVer(version)
+
+      if (semverInstance.major === 0 && semverInstance.minor === 0) {
+        // every one
```

Comment:
Codacy has a fix for the issue: [Insert `····`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263163)
```suggestion
            // every one
```